include/af/array.h

```diff
@@ -1243,11 +1243,12 @@ namespace af
        @{
     */
     inline array &eval(array &a) { a.eval(); return a; }
-    inline void eval(array &a, array &b) { eval(a); b.eval(); }
-    inline void eval(array &a, array &b, array &c) { eval(a, b); c.eval(); }
-    inline void eval(array &a, array &b, array &c, array &d) { eval(a, b, c); d.eval(); }
-    inline void eval(array &a, array &b, array &c, array &d, array &e) { eval(a, b, c, d); e.eval(); }
-    inline void eval(array &a, array &b, array &c, array &d, array &e, array &f) { eval(a, b, c, d, e); f.eval(); }
+    AFAPI void eval(array &a, array &b);
+    AFAPI void eval(array &a, array &b, array &c);
+    AFAPI void eval(array &a, array &b, array &c, array &d);
+    AFAPI void eval(array &a, array &b, array &c, array &d, array &e);
+    AFAPI void eval(array &a, array &b, array &c, array &d, array &e, array &f);
+    AFAPI void eval(int num, array *arrays);
```

Comment:
Would it be bad to have `std::vector<array>` here instead of c-style array since it is a C++ API function ?